po/ar.po

```diff
@@ -4277,73 +4919,102 @@ msgid ""
 
 #: src/user-defined-types/static.md
 msgid ""
 "Because `static` variables are accessible from any thread, they must be "
 "`Sync`. Interior mutability is possible through a [`Mutex`](https://doc.rust-"
 "lang.org/std/sync/struct.Mutex.html), atomic or similar."
 msgstr ""
+"نظرًا لأن المتغيرات الثابتة (`static` variables) يمكن الوصول إليها من أي خيط "
+"(thread)، يجب أن تكون `Sync`. القابلية للتغيير الداخلي (Interior mutability) "
+"ممكنة من خلال [`Mutex`](https://doc.rust-lang.org/std/sync/struct.Mutex."
+"html)، الذري (atomic) أو ما شابه."
 
 #: src/user-defined-types/static.md
 msgid "Thread-local data can be created with the macro `std::thread_local`."
 msgstr ""
+"يمكن إنشاء البيانات المحلية للخيط (Thread-local data) باستخدام الماكرو `std::"
+"thread_local`."
 
 #: src/user-defined-types/const.md
 msgid "`const`"
-msgstr ""
+msgstr "`const`"
 
 #: src/user-defined-types/const.md
 msgid ""
 "Constants are evaluated at compile time and their values are inlined "
 "wherever they are used:"
 msgstr ""
+"يتم تقييم الثوابت (Constants) في وقت الترجمة (compile time) ويتم تضمين قيمها "
+"في أي مكان يتم استخدامها فيه:"
 
 #: src/user-defined-types/const.md
 msgid ""
 "According to the [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-"
 "vs-static.html) these are inlined upon use."
 msgstr ""
+"وفقًا لـ [Rust RFC Book](https://rust-lang.github.io/rfcs/0246-const-vs-"
+"static.html) يتم تضمين هذه القيم عند الاستخدام."
 
 #: src/user-defined-types/const.md
 msgid ""
 "Only functions marked `const` can be called at compile time to generate "
 "`const` values. `const` functions can however be called at runtime."
 msgstr ""
+"فقط الدوال المميزة بـ `const` يمكن استدعاؤها في وقت الترجمة (compile time) "
+"لتوليد قيم `const`. ومع ذلك، يمكن استدعاء الدوال `const` في وقت التشغيل "
+"(runtime)."
 
 #: src/user-defined-types/const.md
 msgid "Mention that `const` behaves semantically similar to C++'s `constexpr`"
-msgstr ""
+msgstr "اذكر أن `const` يتصرف بشكل مشابه دلاليًا لـ `constexpr` في C++."
 
 #: src/user-defined-types/const.md
 msgid ""
 "It isn't super common that one would need a runtime evaluated constant, but "
 "it is helpful and safer than using a static."
 msgstr ""
+"ليس من الشائع جدًا أن يحتاج المرء إلى ثابت يتم تقييمه في وقت التشغيل (runtime "
+"evaluated constant)، ولكنه مفيد وأكثر أمانًا من استخدام ثابت (static)."
 
 #: src/user-defined-types/aliases.md
 msgid ""
 "A type alias creates a name for another type. The two types can be used "
 "interchangeably."
 msgstr ""
+"الاسم المستعار للنوع (type alias) ينشئ اسمًا لنوع آخر. يمكن استخدام النوعين "
+"بالتبادل."
 
 #: src/user-defined-types/aliases.md
 msgid "// Aliases are more useful with long, complex types:\n"
 msgstr ""
+"// الأسماء المستعارة (Aliases) تكون أكثر فائدة مع الأنواع الطويلة والمعقدة:\n"
 
 #: src/user-defined-types/aliases.md
 msgid "C programmers will recognize this as similar to a `typedef`."
-msgstr ""
+msgstr "مبرمجو C سيتعرفون على هذا كشيء مشابه لـ `typedef`."
```

Comment:
This sentence reads weird, maybe its the formatting.

Reply:
yes, the formatting has an issue, we can go through the course and read it in Arabic and make small fixes once we finish